openc3/lib/openc3/models/scope_model.rb

```diff
@@ -350,8 +350,11 @@ def deploy(gem_path, variables)
       # UNKNOWN PacketLog Microservice
       deploy_unknown_packetlog_microservice(gem_path, variables, @parent)
 
-      # Periodic Microservice
-      deploy_periodic_microservice(gem_path, variables, @parent)
+      # Only DEFAULT scope
+      if @scope == 'DEFAULT'
+        # Periodic Microservice
+        deploy_periodic_microservice(gem_path, variables, @parent)
+      end
```

Comment:
I think this makes sense to only be the DEFAULT scope but our 20221210174900_convert_to_multi.rb migration starts the periodic microservice for all scopes. Do we need a migration to stop this for all existing scopes _except_ DEFAULT?

Reply:
That would be better yes.  The extras won't hurt anything, but they are unneeded so we can add a migration to remove them.